green-shop/source/sass/pages/_cart.scss

```diff
@@ -0,0 +1,299 @@
+// ======================================================
+// Pages :: Cart
+// ======================================================
+
+.product-cart {
+  @extend %wrapper;
+
+  &__checkout {
+    text-align: right;
+    margin-bottom: 200px;
```

Comment:
ÍT khi dùng margin to đến vậy, Em tìm hiểu xem cách nào mà tốt cho hiển thị ở tất cả các màn hình
nha

Reply:
Vâng e sẽ chuyển sang dùng vh